<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

  box-shadow: 0px 3px 6px 0px rgba(0, 0, 0, 0.15), 0px 2px 4px 0px rgba(0, 0, 0, 0.12);
  margin-inline-start: auto;
  margin-inline-end: auto;
  margin-block-start: 2em;
  max-inline-size: 22rem;
  overflow: hidden;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
  box-shadow: 0px 3px 6px 0px rgba(0, 0, 0, 0.15), 0px 2px 4px 0px rgba(0, 0, 0, 0.12);
  margin-inline-start: auto;
  margin-inline-end: auto;
  margin-block-start: 2em;
  max-inline-size: 22rem;
  overflow: hidden;
</string>
   </property>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>A/C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Delete">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Split">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Multi">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Plus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Negative">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dot">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Equal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-gap: 1px;
  background: hsl(207, 19%, 61%);

</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Display_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>275</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
  background: hsl(202, 11%, 29%);
  color: hsl(255, 100%, 100%);
  font-size: 2rem;
  padding-block-start: 0.5rem;
  padding-block-end: 0.5rem;
  padding-inline-end: 0.75rem;
  text-align: end;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>275</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
  background: hsl(202, 11%, 29%);
  color: hsl(255, 100%, 100%);
  font-size: 2rem;
  padding-block-start: 0.5rem;
  padding-block-end: 0.5rem;
  padding-inline-end: 0.75rem;
  text-align: end;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <action name="actionOpen_new_Window">
   <property name="text">
    <string>Open new Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
